recipes/cf_xarray/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "cf_xarray" %}
```

How about matching the name on PyPI?

```suggestion
{% set name = "cf-xarray" %}
```

---

Never mind... Please disregard this suggestion. It looks like the [artifacts on PyPI](https://pypi.org/project/cf-xarray/#modal-close) have an '_' in them instead of a `-`.

---

PyPI does some "fuzzing" and both - and _ are accepted. Conda does not! If you want to name it `cf-xarray` and also have a installable `cf_xarray` we need to do a metapackage. But we can do that later in the feedstock.